### ISSUE TITLE
Make ivpn cli work when there's no local connectivity

### ISSUE THREAD

**Issue by vmmello** [ivpn/desktop-app-cli#4]
# Feature request

## Description

* Device: Linux PC, amd64
* OS name and version: Ubuntu 18.04 LTS
* IVPN app version: 2.12.2

I've installed the client on my local network firewall (i.e not my personal computer) and it immediately cut my local network connectivity. I realized that it might have set some restrictive iptables rules. Then I flushed the iptables rules (with: `iptables -F`) and then all subcommands of `ivpn` hang. I tried to fix it by running:

```
# ivpn firewall -off
```
But this command hangs. Even running `# ivpn` hangs. Two minutes later it shows the connection timeout message.

I realize that it hangs because it's unable to connect to the local managent port 127.0.0.1:45429. So it hangs for 2 minutes until the TCP connection times out.

### Steps to reproduce:

Run:
`# iptables -F INPUT`
`# iptables -P INPUT DROP`

Then try running any subcommand of `ivpn` cli.

## Suggested solution

Make the CLI listen for commands on a unix socket (e.g. `/run/ivpn/management.sock` or `/var/run/ivpn/management.sock` for older systems) instead of a TCP socket. This way the command could work even when the local firewall is messed up.

This solution of using a local unix socket instead of a TCP socket is used by many popular software (e.g. Docker, LXC, etc), even for HTTP requests. I guess it's not that difficult to add support for it (or even completely replace the TCP socket with it).

## Making it easier to contribute with pull requests

I guess that this repository is missing some files like `firewall.sh`, `client.up`, `client.down`, `install.sh`, `install-path.sh` and most files from `/usr/share/pleaserun/ivpn-service/`. It also appears that on directory `References/Linux/scripts/` it's missing the `build-all.sh` script.

If you can add those files to this repository it'd be easier to contribute with small suggestions and patches.

**Comment by stenya** [comment]
@vmmello Hi!
We do not have plans to use the Unix sockets yet. But we will consider such a change in the future.
Anyway, did you try to use `-lan_allow` flag for configuring IVPN firewall?
`ivpn firewall -lan_allow`
It can solve your problem with local connectivity.

Regarding missing files:
Files firewall.sh, client.up, client.down are located in 'desktop-app-daemon' repository. CLI project does not use them directly. So, such files should not be located here.
'install' scripts are a part of the generated packages (using FPM). 
The `build-packages.sh` is an analog of `build-all.sh` file you requested.

**Comment by vmmello** [comment]
This command `# ivpn firewall -lan_allow` also hangs when the firewall is messed up. Just like `# ivpn firewall -h` (i.e. I can't even see the help messages for the firewall sub commands).

I'm not blocked by this, I realized how to make it work. I just reported because it took me 20 minutes to realize it. And it blocked my local traffic on the first connect. So I thought it was worth to report it when it's not obvious and happened on the first minutes of usage of the app.
